scripts/build.js

```diff
@@ -23,7 +23,7 @@ database.connect(function (db) {
         // builder basic pokemon details
         const builder = require(__base + 'app/controllers/filler/' + collection);
         builder.insertToDb(function () {
-            return;
+            process.exit();
```

Comment:
@fabe85 because it doesn't work for me. try it yourself. command: npm run build --collection=pokemon